src/SnmpLibrary/library.py

```diff
@@ -249,7 +252,8 @@ def _get(self, oid, idx=(0,), expect_display_string=False):
             self._active_connection.cmd_gen.getCmd(
                 self._active_connection.authentication_data,
                 self._active_connection.transport_target,
-                oid
+                oid,
+                contextName=self._active_connection.context_name,
```

Comment:
please remove the ',' from the end of the line

Reply:
Done in 0f18d150a6ac21aaf7dcf4dfddbf732bcacc3ace